### DISCUSSION TITLE
Debian base build repo/ppa?

### DISCUSSION THREAD

**Discussion by GL513** [Chatterino/chatterino2#3772] [Q&A] [answered]
I currently have the flatpak installed.
I see in the releases that there are .deb files built, but is there a repository or ppa I can add to get updates when I run updates?
When I do `apt search twitch` on Kubuntu 22.04 nothing about chatterino shows up.
Keep up the good work! I may become more involved with the project later on, depending on if I actually use twitch consistently :)

**Comment by pajlada** [comment, marked as answer]
I'd be open to having a PPA for Chatterino. I don't think we have any contributor that uses Debian right now, so any Debian efforts are very welcome.

**Reply by GL513** [reply]
Alright! The thing is, I still need to figure out how to start a ppa. I think canonical hosts all of the Ubuntu ppa-s since they are really just files with simple html sites, but I can't be sure for any other Debian bases.

**Reply by GL513** [reply]
I've got the ppa, I just need to learn how to put a file up there...
https://launchpad.net/~gl513/+archive/ubuntu/ppa-chatterino
